brainstormforce/ultimate-addons-for-gutenberg
# Fix: Advanced Columns - Drag and drop layout broken issue.

### Description

- Added CSS for SVG to resolve the issue

### Screenshots

- Before  - https://share.getcloudapp.com/z8ukwwpY
- After - https://share.getcloudapp.com/9ZuDWWqJ

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Drag and drop advanced column
- Check column layout SVG's in next-release ( enable/disable file generation and regenerate assets delete custom style CSS files and refresh the page to replicate the issue )
- Then switch to `advanced-column-drag-drop-layout-broken-issue` branch to resolve the issue.

### Checklist:
